CHANGELOG.rst

```diff
@@ -1,10 +1,17 @@
 Changelog
 =========
 
-1.0.0 (2020-9-25)
+1.1.0 (2021-3-10)
 -------------------
-*  BREAKING: Remove GC_PROJECT_ID (#183)
+* Google Pubsub 2.0 Compat (#192)
```

Comment:
These changes are break compatibility with pubsub < 2.0 so we should add this right?
```suggestion
* BREAKING Google Pubsub 2.0 Compat (#192)
```

```

Reply:
Ah I see it doesn't break since we force the version